src/status_im2/common/scan_qr_code/view.cljs

```diff
@@ -0,0 +1,239 @@
+(ns status-im2.common.scan-qr-code.view
+  (:require [clojure.string :as string]
+            [reagent.core :as reagent]
+            [oops.core :as oops]            
+            [status-im2.common.scan-qr-code.style :as style]
+            [react-native.camera-kit :as camera-kit]
+            [re-frame.core :as re-frame]
+            [status-im.multiaccounts.key-storage.core :as key-storage]
+            [react-native.core :as rn]
+            [quo2.core :as quo]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [react-native.blur :as blur]
+            [react-native.hole-view :as hole-view]
+            [quo2.foundations.colors :as colors]
+            [status-im.utils.platform :as platform]
+            [react-native.permissions :as permissions]
+            [status-im2.contexts.syncing.utils :as sync-utils]
+            [status-im2.contexts.onboarding.common.background.view :as background]
```

Comment:
this should not be using the onboarding background.

Reply:
Hi @J-Son89  I need to use the same background component, if I don't use this, I will have to copy this component into shell scan which will duplicate the code. Plz suggest

Reply:
Sure, so I believe that background is specific to the onboarding pages - i.e it has a background image from the first page. 

I think in this case it's better to just copy out the blur view from the onboarding background, i.e
```
[blur/view
        {:style         style/background-blur-overlay
         :blur-amount   30
         :blur-radius   25
         :blur-type     :transparent
         :overlay-color :transparent}]
 ```
Because that's all you really need - the onboarding background has extra elements which you won't need on this page 👍

Reply:
Thanks @J-Son89  , I have used it. I earlier used Onboarding background because of figma background (https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?type=design&node-id=5569-526602&mode=design&t=vBjsXfhXjlFUIWe4-0) which has background similar to onboarding.
After discussing with @siddarthkay  , I have now used this blurView & removed onboarding background.